x/tradeshield/keeper/pending_spot_order.go

```diff
@@ -250,9 +249,9 @@ func (k Keeper) ExecuteLimitSellOrder(ctx sdk.Context, order types.SpotOrder) (*
 		Sender:    order.OwnerAddress,
 		Recipient: order.OwnerAddress,
 		Amount:    order.OrderAmount,
```

Comment:
amount for this should be order.OrderAmount * Rate, right ?
because that's what we need to send for swap